x/dclauth/keeper/keeper.go

```diff
@@ -48,3 +48,7 @@ func (k Keeper) MemKey() sdk.StoreKey {
 func (k Keeper) AccountApprovalsCount(ctx sdk.Context) int {
 	return int(math.Round(types.AccountApprovalsPercent * float64(k.CountAccountsWithRole(ctx, types.Trustee))))
 }
```

Comment:
Very important:
1) Add `rejects` field into `types.NewAccount`.
2) Please fix `SetAccount` method in `x/dclauth/keeper/account.go` as in https://github.com/zigbee-alliance/distributed-compliance-ledger/pull/296/files:
```
dclAccO := types.NewAccount(ba, dclAcc.GetRoles(), dclAcc.GetApprovals(), dclAcc.GetRejects(), dclAcc.GetVendorID())
```
Otherwise rejects will be overridden after the given account signs a transaction for the first time.

Reply:
Done! ✅